docs/user/alerting/alerting-troubleshooting.asciidoc

```diff
@@ -199,3 +199,17 @@ In addition to the above methods, broadly used the next approaches and common is
 include::troubleshooting/alerting-common-issues.asciidoc[]
 include::troubleshooting/event-log-index.asciidoc[]
 include::troubleshooting/testing-connectors.asciidoc[]
+
+[discrete]
+[[alerting-kibana-disable]]
+==== Temporarily disable
+
+Cluster performance may become degraded from excessive or expensive {kib} rules. If 
+{kib} is sluggish or unresponsive, you may wish to temporarily reduce load to the
+Task Manager by updating <<task-manager-settings-kb.html|settings>>:
+
+[source, txt]
+--------------------------------------------------
+xpack.task_manager.max_workers: 1
+xpack.task_manager.poll_interval: 1m
```

Comment:
Not sure 1m will be enough, but I've not used this myself, and it kind of depends on the situation.  If an expensive rule ends up "slipping through" (1 task will run every 1m), it could still run for longer than 1m, and affect performance during that period.

I think `1h` is probably more appropriate.

Is noting this as "temporary" enough of a warning here, to remind the reader that Kibana won't really be operating correctly (especially alerting, but other systems use task manager) until the settings revert back to the default.  Seems like we need something to that affect, **in bold** :-)

Reply:
Per Patrick's comment, it seems like this value is preferrable:
```suggestion
xpack.task_manager.poll_interval: 1h
```

Reply:
> Is noting this as "temporary" enough of a warning here, to remind the reader that Kibana won't really be operating correctly (especially alerting, but other systems use task manager) until the settings revert back to the default. Seems like we need something to that affect, **in bold** :-)

I've added a "warning" section via a7d3570f32dcb7f965b02778d6e4f93e69314495 to reinforce that this is a last-resort and temporary option.